code/modules/food_and_drinks/kitchen_machinery/deep_fryer.dm

```diff
@@ -84,26 +86,34 @@
 		emagged = TRUE
 		return
 
-/obj/machinery/cooker/deepfryer/special_attack(obj/item/grab/G, mob/user)
-	if(ishuman(G.affecting))
-		if(G.state < GRAB_AGGRESSIVE)
-			to_chat(user, "<span class='warning'>You need a better grip to do that!</span>")
-			return 0
-		var/mob/living/carbon/human/C = G.affecting
-		var/obj/item/organ/external/head/head = C.get_organ("head")
-		if(!head)
-			to_chat(user, "<span class='warning'>This person doesn't have a head!</span>")
-			return 0
-		C.visible_message("<span class='danger'>[user] dunks [C]'s face into [src]!</span>", \
-						"<span class='userdanger'>[user] dunks your face into [src]!</span>")
-		C.emote("scream")
-		user.changeNext_move(CLICK_CD_MELEE)
-		C.apply_damage(25, BURN, "head") //25 fire damage and disfigurement because your face was just deep fried!
-		head.disfigure()
-		add_attack_logs(user, G.affecting, "Deep-fried with [src]")
-		qdel(G) //Removes the grip so the person MIGHT have a small chance to run the fuck away and to prevent rapid dunks.
-		return 0
-	return 0
+/obj/machinery/cooker/deepfryer/special_attack_shove(mob/living/target, mob/living/attacker)
+	. = ..()
+	target.visible_message(
+		"<span class='danger'>[attacker] shoves [target] against [src], and [target] reaches into the hot oil trying to catch [target.p_their()] fall!</span>",
+		"<span class='userdanger'>[attacker] shoves you into [src], your hands landing in hot oil!</span>",
+		"<span class='danger'>You hear a splash and a loud sizzle.</span>"
+	)
+	target.apply_damage(2, BURN, BODY_ZONE_PRECISE_L_HAND)
+	target.apply_damage(2, BURN, BODY_ZONE_PRECISE_R_HAND)
+	playsound(src, 'sound/goonstation/misc/drinkfizz.ogg', 25)
+	return FALSE
```

Comment:
4 damage for getting shoved in oil is pitiful. punching someone does more. imo it should be atleast 20